src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/begrunnelser/EØSStandardbegrunnelse.kt

```diff
@@ -342,6 +342,10 @@ enum class EØSStandardbegrunnelse : IVedtakBegrunnelse {
         override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
         override val sanityApiNavn = "innvilgetSelvstendigRettTilleggstekstSekundarInfoOmMuligRefusjon"
     },
+    INNVILGET_SELVSTENDIG_RETT_SEKUNDÆR_ANNEN_FORELDER_BOR_I_NORGE {
+        override val vedtakBegrunnelseType = VedtakBegrunnelseType.EØS_INNVILGET
+        override val sanityApiNavn = "innvilgetSelvtsendigRettSekundaerAnnenForelderBorINorge"
```

Comment:
```suggestion
        override val sanityApiNavn = "innvilgetSelvstendigRettSekundaerAnnenForelderBorINorge"
```

Burde rettes opp i sanity også